### ISSUE TITLE
model.eval_metrics changes metric names in the output

### ISSUE THREAD

**Issue by avanunts** [catboost/catboost#2683]
Problem: call of a method `eval_metrics` of `CatBoostRanker` class returns a dict with keys that are different from keys in the argument `metrics` of the method. 

catboost version: 1.2.5
Operating System: NAME="Ubuntu", VERSION="18.04.6 LTS (Bionic Beaver)"
CPU: Intel(R) Xeon(R) Gold 6248R CPU @ 3.00GHz
GPU: NVidia A40

Minimal example: 
```
catboost_ndcg_score = model.eval_metrics(
    data=test_pool.slice(diverged_group_index), 
    metrics=['NDCG:top=24;denominator=LogPosition']
)
print(catboost_ndcg_score)
```

Expected output: 
```
{'NDCG:top=24;denominator=LogPosition':  <list of metric for iterations>}
```

Got output: 
```
{'NDCG:top=24;type=Base': <list of metric for iterations>}
```

The values are correct.

**Comment by avanunts** [comment]
Addition. 

Another example: 

```
catboost_ndcg_score = model.eval_metrics(
    data=test_pool.slice(diverged_group_index), 
    metrics=['NDCG:top=24;denominator=Position']
)

print(catboost_ndcg_score)
```

outputs 

`{'NDCG:top=24;type=Base': <list of metric for iteration>}`

It appears that the `denominator` argument is simply not written in the name of metric for some reason.

**Comment by avanunts** [comment]
The values are correct again.

**Comment by avanunts** [comment]
And one more: 

```
catboost_ndcg_score = model.eval_metrics(
    data=test_pool.slice(diverged_group_index), 
    metrics=['NDCG:top=24;denominator=Position', 'NDCG:top=24;denominator=LogPosition'], 
    ntree_start=model.tree_count_ - 1, 
    ntree_end=model.tree_count_
)

print(catboost_ndcg_score)
```

outputs 
```
{'NDCG:top=24;type=Base': <...>}
```

The value if for `LogPosition`.

```
catboost_ndcg_score = model.eval_metrics(
    data=test_pool.slice(diverged_group_index), 
    metrics=['NDCG:top=24;denominator=LogPosition', 'NDCG:top=24;denominator=Position'], 
    ntree_start=model.tree_count_ - 1, 
    ntree_end=model.tree_count_
)

print(catboost_ndcg_score)
```

Outputs a dict with the same key, but the value is for `Position`.

**Comment by ek-ak** [comment]
Hello!
Thank you for bug report. Denominator will be in the metric description in the next release.

**Comment by Evgueni-Petrov-aka-espetrov** [comment]
fixed in 844b1663121b6fda9f8290be5a29f8a1682aa1fb
